### DISCUSSION TITLE
Being able to have custom headers for compressed files.

### DISCUSSION THREAD

**Discussion by CSEliot** [yandeu/five-server#105] [Help / Questions]
Anything compressed (GZip, Brotli) needs the webserver to have in it's header a specific line. See error from firefox:
**Unable to parse Build/MJ2CG.framework.js.gz! This can happen if build compression was enabled but web server hosting the content was misconfigured to not serve the file with HTTP Response Header "Content-Encoding: gzip" present. Check browser Console and Devtools Network tab to debug.**

Need to be able to add code like this:
https://docs.unity3d.com/2020.1/Documentation/Manual/webgl-server-configuration-code-samples.html

or add this (content encoding headers):
https://docs.unity3d.com/2020.1/Documentation/Manual/webgl-deploying.html

Thanks in advance for your time!

**Comment by yandeu** [comment]
You should be able to add your custom header using a expressjs like middleware in your config.js file.

See https://github.com/yandeu/five-server/discussions/76
